src/backend/opencl/cl/cn/algorithm.cl

```diff
@@ -24,6 +24,7 @@
 #define ALGO_RX_SFX         0x72151273
 #define ALGO_RX_KEVA        0x7214116b
 #define ALGO_RX_GRAFT       0x72151267
+#define ALGO_RX_LOZZ        0x7213f747
```

Comment:
Should be `0x7215126c`...

Because `RANDOMX_SCRATCHPAD_L3` is `2097152`, thus `Log2(2097152) = 21` which is `0x15`
And `RANDOMX_SCRATCHPAD_L2` is `262144`, thus `Log2(262144) = 18` which is `0x12`
And the ASCII hex byte for the first letter of the name (lowercase `L`) is `0x6c`

So `0x7215126c`

The L3/L2 sizing bytes are used in some of the code and must be correct.  Not even sure what it would do with `2^0xf7` sized L2, lol.  Where did `0xf747` come from?

Reply:
Unless, `0x13` was intentional for a `RANDOMX_SCRATCHPAD_L3` of `524288`, in which case it's incorrect over in the [`src/backend/opencl/cl/rx/randomx_constants_lozz.h`](https://github.com/xmrig/xmrig/blob/46a82c25e913a27056871254daec3b7396229869/src/backend/opencl/cl/rx/randomx_constants_lozz.h)

`0xf7` still makes no sense either way because there aren't `2.261564243×10⁷⁴` bytes of L2 cache in total on the planet.

Reply:
Resolved [#3c7017b](https://github.com/xmrig/xmrig/pull/2875/commits/3c7017be8acba15bfdde346232399c9743c6a1f1)